bindings/python/src/utils/pretokenization.rs

```diff
@@ -272,10 +270,10 @@ impl DestroyPtr for PyPreTokenizedStringRefMut {
 }
 
 impl PyPreTokenizedStringRefMut {
-    pub fn new(pretok: &mut tk::PreTokenizedString) -> RefMutGuard<Self> {
+    pub fn new(pretok: &mut tk::PreTokenizedString) -> super::RefMutGuard<'_, Self> {
         // SAFETY: This is safe because we return a RefMutGuard here.
         // The compiler will make sure the &mut stays valid as necessary.
-        RefMutGuard::new(Self {
+        super::RefMutGuard::new(Self {
```

Comment:
any reason not to import it from super directly?

Reply:
Commit slipped in from a different PR I was working on, it's an immaterial change, undone.

Reply:
When making refactors, I find it helps the readability of changes to have diffs that are as localized as possible. If subsequent commits touch these lines, I might move the imports to be more local.